build.gradle

```diff
@@ -892,15 +892,14 @@ task packagePulse {
     pulseDir = "${projectDir}/../pulse"
   }
 
-  if (!file(pulseDir).canWrite()){
-    throw  new Exception(" Pulse directroy can not be empty while building product")
-  }
-
-  println "Using Pulse location ${pulseDir}"
+  doFirst {
+    if (!file(pulseDir).canWrite()){
+      throw  new Exception(" Pulse directroy can not be empty while building product")
```

Comment:
The message should be corrected to something like: s"Pulse checkout directory $pulseDir not writable"